src/ts/interfaces/ITelemetryData.ts

```diff
@@ -40,6 +40,7 @@ export enum TYPERIGHTER_TELEMETRY_TYPE {
   TYPERIGHTER_MARK_AS_CORRECT = "TYPERIGHTER_MARK_AS_CORRECT",
   TYPERIGHTER_MATCH_FOUND = "TYPERIGHTER_MATCH_FOUND",
   TYPERIGHTER_CHECK_DOCUMENT = "TYPERIGHTER_CHECK_DOCUMENT",
+  TYPERIGHTER_CLEAR_DOCUMENT = "TYPERIGHTER_CLEAR_DOCUMENT",
```

Comment:
V. minor – I'd recommend this (and other things further down, e.g. telemetry methods) be called 'clear matches' or similar – just because it's likely that in future we're clearing matches from things we might not call a document, e.g. headline, image caption etc.

Reply:
I was aiming to be consistent with this naming as check and clear are antonyms in this context, so you would either be checking for matches or clearing matches. Therefore it made sense to me to name it this way. I'm happy to change it, but would we want to change the "TYPERIGHTER_CHECK_DOCUMENT" name too?

Reply:
Haha, good point. Let's leave it for now – changing `CHECK_DOCUMENT`  would mean an inconsistency in our telemetry, and we know what we mean 😁